dlt/pipeline/__init__.py

```diff
@@ -99,6 +107,8 @@ def pipeline(
     import_schema_path: str = None,
     export_schema_path: str = None,
     full_refresh: bool = False,
```

Comment:
Imho we should set full_refresh to "None" and issue a warning if it set to True or False explicitely. So:
* If full_refresh is None, take the value of dev_mode
* If full_refresh is not None, issue warning and set dev_mode to full_refresh

Reply:
@rudolfix wdyt

Reply:
That's a good approach. Distinguish between `False` and "not used"